### DISCUSSION TITLE
API // Lastmanagement

### DISCUSSION THREAD

**Discussion by GodHunter** [goecharger/go-eCharger-API-v2#235] [Q&A]
Hallo zusammen,

ich habe heute den 2. go e Charger in Betrieb genommen.
Bislang habe ich den einen via API dynamisch für den Stromüberschuss gesteuert was auch echt gut funktionierte.

Mit dem 2. tue ich mich jetzt schwer, bzw. weiß ich nicht wie ich das über das Lastmanagement tun könnte.
Gibt es die Möglichkeit dem Lastmanagement per API zu sagen wie viel Ampere derzeit verfügbar sind und das Lastmanagement verteilt es dann auf die in der Gruppe definierten Charger?

Oder muss ich mir hier quasi ein eigenes Lastmanagement basteln?

Im API Status des Chargers taucht ja z.B. das hier auf:

`"lot":{"amp":16,"dyn":16,"sta":16,"ts":1721902915}`

Könnte ich da nicht den Wert bei dynamisch ( dyn ) einfach verändern und den Rest machen die beiden Wallboxen unter sich aus?

**Comment by markus183** [comment]
Hast du schon versucht ids={} an beide Charger schicken? Je nach Einstellung des Lastmanagements werden die Charger den vorhandnen pv Strom verwenden.
